src/sass/app/metatrader.scss

```diff
@@ -340,6 +340,12 @@
                     font-weight: bold;
                     color: $COLOR_GRAY_SHADE;
                 }
+                #insufficient_funds {
+                    a {
```

Comment:
possible to add a class name to avoid using general css statements on this?

Reply:
added a class for the anchor now